assets/css/style.css

```diff
@@ -0,0 +1,206 @@
+/* reseting things */
+h1,
+h2,
+h3,
+h4,
+h5,
+h6 {
+  color: white;
+  font-weight: normal;
+  margin: 0 0 5px 0;
+}
+
+body {
+  font-family: 'robotothin', Arial, Helvetica, sans-serif;
+  margin: 0;
+}
+
+/* font setting */
+
+@font-face {
+  font-family: 'robotothin';
+  font-style: normal;
+  font-weight: lighter;
+  src:
+    url(../fonts/roboto-thin-webfont.woff) format('woff'),
+    url(../fonts/roboto-thin-webfont.woff2) format('woff2');
+}
+
+@font-face {
+  font-family: 'robotoregular';
+  font-style: normal;
+  font-weight: lighter;
+  src:
+    url(../fonts/roboto-regular-webfont.woff) format('woff'),
+    url(../fonts/roboto-regular-webfont.woff2) format('woff2');
+}
+
+/* general settings */
+
+a {
+  color: #20a9d3;
+  text-decoration: none;
+}
+
+a:hover {
+  text-decoration: underline;
+}
+
+.footer-link li {
+  border-right: 1px solid #cdcdcd;
+  padding: 0 4px;
+}
+
+.list-inline li {
+  display: inline-block;
+}
+
+/* nav bar */
+
+.nav-container {
+  background: linear-gradient(to bottom, #8c8c8c 0%, #555 50%, #7c7c7c 52%, #a8a8a8 100%);
```

Comment:
try with this gradient instead do you think it looks more like the original? :thinking: 

`background: linear-gradient(to bottom,gray 0%,#7c7c7c 50%,#666 51%,gray);`

Reply:
Yeah! It looks more like the original, thank you again.